sqlitecluster/SQLite.cpp

```diff
@@ -760,6 +779,12 @@ int SQLite::commit() {
     // If there were conflicting commits, will return SQLITE_BUSY_SNAPSHOT
     SASSERT(result == SQLITE_OK || result == SQLITE_BUSY_SNAPSHOT);
     if (result == SQLITE_OK) {
+        if (_currentTransactionAttemptCount != -1) {
+            string logLine = SWHEREAMI + "[row-level-locking] transaction attempt:" +
```

Comment:
this is confusing to me. this says "transaction attempt" which to me implies it's the attempt number for a given transaction, but the number increases forever until the binary is restarted. so it's not actually the attempt for this transaction.

Reply:
It's the attempt for all transactions, like `commitCount`, but including failed commits.